src/frontends/onnx/tests/onnx_reader_external_data.cpp

```diff
@@ -10,13 +10,17 @@
 #include <fstream>
 #include <ie_core.hpp>
 #include <ngraph/ngraph.hpp>
+#include <openvino/frontend/manager.hpp>
+#include <openvino/openvino.hpp>
```

Comment:
- let's use "" for OV headers and <> only for system ones.
- better to avoid global headers (like openvino/openvino.hpp) in our code to minimize compile time and rebuild